libbeat/docs/shared-path-config.asciidoc

```diff
@@ -106,3 +106,20 @@ Example:
 ------------------------------------------------------------------------------
 path.logs: /var/log/beats
 ------------------------------------------------------------------------------
+
+[float]
+==== `system.hostfs`
+
+Specifies the mount point of the host's filesystem for use in monitoring a host.
+This can either be set in the config, or with the `--system.hostfs` CLI flag. This is used for cgroup self-monitoring.
+ifeval::["{beatname_lc}"=="metricbeat"]
```

Comment:
If this setting is used for other beats too, I think we could add it as `path.hostfs` for the config file setting, as other beats don't have a `system` module. But we can leave it for a future change.